stanford_sites_helper.module

```diff
@@ -50,7 +50,7 @@ function stanford_sites_helper_search_noresults() {
  * Implements hook_block_info().
  */
 function stanford_sites_helper_block_info() {
-  if (stanford_sites_helper_stanford_sites_hosted()) {
+  if (stanford_sites_helper_stanford_sites_hosted() || stanford_sites_helper_is_acquia_hosted()) {
```

Comment:
If the help block isn't specific to stanford_sites any more why not just remove this check?

Reply:
I'm trying to remember the logic behind doing that. I guess in case someone installed the module elsewhere and people were submitting tickets for sites not on our infrastructure.

Reply:
🤔 I think I am leaning towards removing the check as technically ACSF specific stuff should go in the acsf_helper module not here. I get why you did this but you are mixing the purpose of the module with another.

Reply:
Yeah, we have a lot of overlap between those two modules. :/

Reply:
Given that it took me entirely too much time to determine why this block wasn't showing up, I think you've convinced me.